_pytest/python.py

```diff
@@ -424,12 +429,17 @@ def _importtestmodule(self):
                  % e.args
             )
         except ImportError:
-            exc_class, exc, _ = sys.exc_info()
+            from _pytest._code.code import ExceptionInfo
+            exc_info = ExceptionInfo()
+            if self.config.getoption('verbose') < 2:
+                exc_info.traceback = exc_info.traceback.filter(filter_traceback)
+            exc_repr = exc_info.getrepr(style='short') if exc_info.traceback else exc_info.exconly()
```

Comment:
Decided to hard-code it to `short` because it is usually enough to track import problems and `long` may generate too much output during collection.